ch.sbb.scion.rcp.microfrontend/src/ch/sbb/scion/rcp/microfrontend/browser/BrowserViewType.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Project: RCS - Rail Control System
+ *
+ * © Copyright by SBB AG, Alle Rechte vorbehalten
+ */
+package ch.sbb.scion.rcp.microfrontend.browser;
+
+/**
+ * The list of supported {@link BrowserView} types.
+ */
+public enum BrowserViewType {
+  /**
+   * JxBrowser's SWT browser view control.
+   */
+  JXBROWSER,
+  /**
+   * Standard Eclipse SWT browser view control.
+   */
+  SWT
```

Comment:
Why do we need 1) the enum and 2) the factory class for a simple decision to creating one thing or another? 

Can we simplify it to just having two static factory methods in the classes extending `AbstractBrowserView`?

Reply:
We want developers to work with a single type (BrowserView) and be able to switch between implementations easily. If developers use a static factory method of a specific implementation, they will have to import it, which increases the risk of using implementation-specific code. This approach is somewhat error-prone, as the code becomes tied to a specific implementation, making it difficult to switch to another one.

Reply:
> We want developers to work with a single type (BrowserView) 
And you have three things:
1. `BrowserView`
2. `BrowserViewType`
3.  `BrowserViewFactory`.

Why cannot we just have `BrowserView.createStdBrowserView()` and `BrowserView.createJxBrowserView()`?

Reply:
Or have the factory with just to methods. Do not introduce the `enum` which simply duplicates the idea of having just two possible implementations. Why bother?

Reply:
OK. @Dmytro-Fedoryshyn let's remove then `enum` and have the factory with two methods.